src/modules/displayfood.js

```diff
@@ -36,6 +36,7 @@ export const display = async () => {
               </div>
               <div class="column flex">
                 <button class="button">Comments</button>
+                <button class="button">Reservations</button>
             </section>`;
 
         // renderLikes(item.idMeal);
```

Comment:
Same issue as discussed above. I suggest that you do not commit old pieces of code as inline comments.